contrib/opentimelineio_contrib/adapters/tests/test_aaf_adapter.py

```diff
@@ -848,6 +852,20 @@ def test_external_reference_from_unc_path(self):
             unc_path
         )
 
+    def test_external_reference_paths(self):
+        timeline = otio.adapters.read_from_file(COMPOSITE_PATH)
+        for video_track in timeline.video_tracks():
+            for clip in video_track:
+                self.assertIsInstance(clip.media_reference,
+                                      otio.schema.ExternalReference)
+                self.assertIsNotNone(clip.media_reference.target_url)
```

Comment:
If you compare the actual contents of `target_url` then this test will cover the handling of backslashes as well. I have a slight worry that there might be a platform dependency on the path separator in these UNC paths, so it would be good to check that.

Reply:
@JoshBurnell would you mind adding a check for the correct `target_url` here?